tutorials/13_diagnostics_simulation_based_calibration.ipynb

```diff
@@ -541,13 +984,74 @@
   },
   {
    "cell_type": "markdown",
+   "id": "c2b021c4-c1aa-433d-95c4-e09e926f172f",
    "metadata": {},
    "source": [
     "The above provides a visual representation of the cumulative density function (CDF) of `ranks` (blue and orange for each dimension of `theta`) with respect to the 95% confidence interval of a uniform distribution (grey).\n"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "b3f1a0cc-d83f-4f0c-b174-ca367aab7699",
+   "metadata": {},
+   "source": [
+    "## multi dimensional SBC\n",
+    "\n",
+    "So far, we have performed the SBC checks for each dimension of our parameters $\\theta$ separately. SBI offers a way to perform this check for all dimensions at once. For this, we need to scale up our toy example a bit. So we increase the problem dimensionality to `16`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c4de299-54a2-4ebd-beb0-344e90767c9f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "c56e48670ecb47309f927ff55fd6df2b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Running 1000 sbc samples.:   0%|          | 0/1000 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'ks_pvals': tensor([0.1461]), 'c2st_ranks': tensor([0.5555]), 'c2st_dap': tensor([0.5100, 0.4925])}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# use reduce_fns in order to map the test statistic to only one dimension\n",
+    "ranks, dap_samples = run_sbc(\n",
+    "    thetas,\n",
+    "    xs,\n",
+    "    posterior,\n",
+    "    num_posterior_samples=num_posterior_samples,\n",
+    "    reduce_fns=posterior.log_prob,\n",
+    ")\n",
+    "check_stats = check_sbc(ranks, thetas, dap_samples, 1_000)\n",
+    "print(check_stats)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a283036-93e2-44ff-ac1d-fe42ceaaf6a0",
+   "metadata": {},
+   "source": [
+    "In the code above, the `run_sbc` method applies the `posterior.log_prob` on the `{thetas, xs}` provided in the function call. In addition `run_sbc` uses the `reduce_fns` parameter internally on SBC generated `{theta, x}` pairs. In this way, the rank computation is performed on the sum of log probability values obtained from each dimension of theta. SBC can now be performed in one dimenions and hence is easier to grasp in practice. Moreover, the results of `check_sbc` using ranks including the `reduce_fns` parameter comply with those omitting this: ks_pvals is above `0.05` and `c2st_ranks` is about `0.5`."
```

Comment:
have the `reduce_fns` been introduced somewhere above? If not, it would be good to briefly explain what they do. 
More generally, I find it difficult to follow this paragraph. Maybe we can say sth like: 
- so far, we calculated the SBC ranks of the parameters by simply ranking them with respect to their actual value, e.g., `theta_i[idx] < theta_o[idx]`. Thus, we were ranking each parameter only within its *marginal* dimension `idx`.
- The `reduce_fn` argument allows us to specify a `Callable` to rank `theta` with respect to all dimensions of `theta`. E.g., we can specify `reduce_fn=posterior.log_prob` to rank $\theta_i$ according to its log probability under the posterior - across all dimensions of `theta`. 

I am not sure I understand what you want to say with the last sentence in this paragraph. Can you clarify please?

Reply:
Sounds good to me. I'll see to adding that.

Reply:
fixed with b5c82ca0